src/components/FullscreenGrid/index.tsx

```diff
@@ -18,54 +28,14 @@ export const Header = ({
   );
 };
 
-export type PanelAttribs = 'altSidePanel' | 'mainSidePanel';
-
-type ContentShellProps = {
-  altSidePanel?: AnyReactNode;
-  altSidePanelClass?: string;
-  children?: AnyReactNode;
-  className?: string;
-  mainSidePanel?: AnyReactNode;
-  mainSidePanelClass?: string;
-};
-
-export const ContentShell = ({
-  altSidePanel,
-  altSidePanelClass,
-  children,
-  className,
-  mainSidePanel,
-  mainSidePanelClass,
-}: ContentShellProps) => {
-  return (
-    <React.Fragment>
-      <aside
-        className={makeClassName(styles.mainSidePanel, mainSidePanelClass)}
-      >
-        {mainSidePanel}
-      </aside>
-      <main className={makeClassName(styles.content, className)}>
-        {children}
-      </main>
-      <aside className={makeClassName(styles.altSidePanel, altSidePanelClass)}>
-        {altSidePanel}
-      </aside>
-    </React.Fragment>
-  );
-};
+export const ContentShell = ContentShellComponent;
```

Comment:
I find this a bit odd. Why not just make `ContentShell` a component in its own right, as we've done with basically every other component. I realize that we don't ever imagine using it outside of a `FullscreenGrid`, but it still feels weird to have a component which is only accessible from within another component.

What's the advantage to doing it this way?

Reply:
mmh, this was mostly for backward compatibility. Let's move this component to `src/components` then.